src/posts/2020-04-13-simple-but-powerful-pratt-parsing.djot

```diff
@@ -390,7 +390,7 @@ Our current right priority is `2`, and, to be able to fold the expression, we ne
 So let's recursively call `expr_bp` starting at `b`, but also tell it to stop as soon as `bp` drops below `2`.
 This necessitates the addition of `min_bp` argument to the main function.
 
-And lo, we have a fully functioning minimal Prat parser:
+And so, we have a fully functioning minimal Prat parser:
```

Comment:
```suggestion
And lo, we have a fully functioning minimal Prat parser:
```

this one is intentional, lo as in "lo and behold"

Reply:
Oh 😅